src/main/groovy/edu/oregonstate/mist/api/jsonapi/GenericExceptionMapper.groovy

```diff
@@ -0,0 +1,32 @@
+package edu.oregonstate.mist.api.jsonapi
+
+import edu.oregonstate.mist.api.Resource
+import edu.oregonstate.mist.api.Error
+import org.slf4j.Logger
+import org.slf4j.LoggerFactory
+
+import javax.ws.rs.core.Response
+import javax.ws.rs.ext.ExceptionMapper
+import javax.ws.rs.ext.Provider
+
+/**
+ * Created by georgecrary on 8/5/16.
+ */
```

Comment:
Your name and the date are already attached to the commit. It isn't necessary to pepper source files with these kind of comments.